src/My_Experiences.html

```diff
@@ -46,15 +38,11 @@ <h5>Title description, Oct 1, 2022</h5>
     <br>
 <div class="card border-dark mb-3" style="max-width: 60%">
     <div class="card-header"><h2>TITLE HEADING 3</h2></div>
-
     <h5>Title description, Oct 1, 2022</h5>
     <div class="fakeimg" style="height:200px;">Image</div>
     <p>Some text..</p>
 </div>
-    <br>
-
-
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure